packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -386,20 +391,17 @@ function claimFromProofResult(
   if (!babyJubIsNegativeOne(publicSignals[8])) {
     partialTicket.ticketCategory = parseInt(publicSignals[8]);
   }
-  // These three fields are currently not typed or being used, but are kept
-  // as reserved fields that are hardcoded to zero and included in the preimage
-  // of the hashed signature. As such, the flags for revealing these reserved
-  // signed fields should always be -1 until they are being typed and used.
   if (!babyJubIsNegativeOne(publicSignals[9])) {
```

Comment:
Ditto on the explanation of "reserved" here.

Reply:
I've rewritten the comment to refer to the single remaining reserved field.